Source/WebCore/html/HTMLSlotElement.idl

```diff
@@ -30,7 +30,7 @@
     [CEReactions=NotNeeded, Reflect] attribute DOMString name;
     sequence<Node> assignedNodes(optional AssignedNodesOptions options);
     sequence<Element> assignedElements(optional AssignedNodesOptions options);
-    [EnabledBySetting=ImperativeSlotAPIEnabled] undefined assign(Node... nodes); // FIXME: This should be (Element or Text) instead of Node.
+    [EnabledBySetting=ImperativeSlotAPIEnabled] undefined assign((Element or Text)... nodes);
```

Comment:
Can we add a set for this? In theory, we should be throwing in cases like this:
```js
const slot = document.createElement('slot');
slot.assign(document.createComment('foo'));
```

Reply:
I added the test case to wpt.
https://github.com/web-platform-tests/wpt/pull/35443